src/main/java/com/google/sps/servlets/AddEventServlet.java

```diff
@@ -40,10 +40,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     // Only Moderators can add Events.
-    // if (user.isModeratorOfAnyOrg()) {
-    //   // TODO: Can Maintainers add Events?
-    //   throw new IllegalArgumentException("ERROR: User does not have the permission to perform addition of Events.");
-    // }
+    if (!user.isModeratorOfAnyOrg()) {
```

Comment:
should this be !user.isMod && !isMaintainer?

Reply:
yes- will update